lib/two_factor_authentication/models/two_factor_authenticatable.rb

```diff
@@ -87,6 +103,19 @@ def generate_totp_secret
           ROTP::Base32.random_base32
         end
 
+        def create_backup_codes(options = {})
```

Comment:
What do you think about putting this in its own module, like the `devise-two-factor` gem? https://github.com/tinfoil/devise-two-factor/blob/master/lib/devise_two_factor/models/two_factor_backupable.rb